static/js/utils/place_utils.ts

```diff
@@ -196,6 +197,26 @@ export function getPlaceNames(
     });
 }
 
+/**
+ * Given a list of place dcids, returns a promise with a map of dcids to place
+ * display names (adds state code to the end of the name if available)
```

Comment:
Please clarify the difference between `getPlaceNames()` and `getPlaceDisplayNames()`

Reply:
done